test/infrastructure/docker/main.go

```diff
@@ -63,7 +63,7 @@ func main() {
 	flag.IntVar(&concurrency, "concurrency", 10, "The number of docker machines to process simultaneously")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
-	flag.DurationVar(&syncPeriod, "sync-period", 10*time.Minute,
+	flag.DurationVar(&syncPeriod, "sync-period", 3*time.Minute,
```

Comment:
Can we set this somewhere else instead of changing the default?

Reply:
This could prove difficult, because currently clusterctl init does not support changing those values.
However, let's make some test run here before